app/products/calls/components/current_call_bar/current_call_bar.tsx

```diff
@@ -143,6 +145,14 @@ const CurrentCallBar = ({
 
     const micPermissionsError = !micPermissionsGranted && !currentCall?.micPermissionsErrorDismissed;
 
+    // The user should receive an alert if all of the following conditions apply:
+    // - Recording has started.
+    // - Recording has not ended.
+    // - The alert has not been dismissed already.
+    if (currentCall?.recState?.start_at && !currentCall?.recState?.end_at && !currentCall?.recAcknowledged) {
+        recordingAlert(intl);
+    }
```

Comment:
Would the host be receiving the alert as well? I guess the broader question is whether we should be consistent and have the two versions (host and participant), just like one web (and potentially the "recording is processing" as well). I think the answer should be yes but I am not going to enforce this due to the strict timing we are facing here /cc @tanmay-des

Reply:
@streamer45 Yep, that's in: https://github.com/mattermost/mattermost-mobile/pull/6787 -- along with the recording is processing

Reply:
Oh well, I blame jetlag but I think I totally missed the split. Thanks :+1: